README.md

```diff
@@ -1,55 +1,51 @@
+|[TimeSlider](docs/time-slider.md) | ![TimeSlider on UWP](https://user-images.githubusercontent.com/29742178/147712751-6d6db182-3e72-4dfc-ba23-3fbe97b1f934.png) | Interactively manipulate or animate the time extent for a map or scene. | WinUI, UWP, WPF |
+|[UtilityNetworkTraceTool](docs/un-trace.md) | ![Utility Network Trace Tool on WPF](https://user-images.githubusercontent.com/29742178/173907265-73cd3a39-c836-433e-baf0-4c60f921ba86.png) | Use named trace configurations defined in a web map to perform connected trace operations and compare results. | WPF, UWP, WinUI, MAUI |
 
 ## Get started
 
-The simplest way to get started is to add the NuGet package(s) to your projects:
+> **Note**: At version 200.0, Toolkit assemblies are now built individually for each platform. Please replace any existing `Esri.ArcGISRuntime.Toolkit` NuGet reference with Toolkit NuGet package specific to that platform. For example if you have a WPF application using Toolkit, you will need to uninstall `Esri.ArcGISRuntime.Toolkit` NuGet package and add `Esri.ArcGISRuntime.Toolkit.WPF` NuGet package.
+
+The simplest way to get started is to add the platform-specific NuGet package(s) to your projects:
+
 
-[![Link: Toolkit NuGet - base package](https://img.shields.io/badge/Toolkit-007ac2?style=flat-square&labelColor=gray&logo=nuget)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit/)
-[![Link: Toolkit NuGet - Xamarin.Forms](https://img.shields.io/badge/Xamarin.Forms%20Toolkit-007ac2?style=flat-square&labelColor=gray&logo=nuget)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.Xamarin.Forms/)
-[![Link: Toolkit NuGet - AR](https://img.shields.io/badge/Augmented%20Reality%20Toolkit-007ac2?style=flat-square&labelColor=gray&logo=nuget)](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit/)
-[![Link: Toolkit NuGet - Forms AR](https://img.shields.io/badge/Xamarin.Forms%20Augmented%20Reality%20Toolkit-007ac2?style=flat-square&labelColor=gray&logo=nuget)](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit.Forms/)
+[![Link: Toolkit NuGet - WPF](https://img.shields.io/badge/WPF-007ac2?style=flat-square&labelColor=gray&logo=windowsxp)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.WPF/)
+[![Link: Toolkit NuGet - WinUI](https://img.shields.io/badge/WinUI-007ac2?style=flat-square&labelColor=gray&logo=windows11)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.WinUI/)
+[![Link: Toolkit NuGet - UWP](https://img.shields.io/badge/UWP-007ac2?style=flat-square&labelColor=gray&logo=windows)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.UWP/)
+[![Link: Toolkit NuGet - MAUI](https://img.shields.io/badge/MAUI-007ac2?style=flat-square&labelColor=gray&logo=dotnet)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.Maui/)
```

Comment:
These are links to nuget packages, not platforms. I don't get why these badges are changed?